src/couch_mrview/test/couch_mrview_ddoc_updated_tests.erl

```diff
@@ -47,12 +46,17 @@ setup() ->
     {ok, _} =
         couch_mrview:query_view(Db2, <<"_design/bar">>, <<"baz">>, [], CB, 0),
 
+    meck:expect(couch_index_updater, update, fun(Idx, Mod, IdxSt) ->
```

Comment:
Granted in this particular case the point is to never hit the passthrough call but we should try and be correct in our usage of meck or the bad patterns might get copied to other tests if this is used as a reference.